sdk/servicebus/azure-servicebus/azure/servicebus/amqp/_amqp_message.py

```diff
@@ -87,32 +87,32 @@ class AmqpAnnotatedMessage(object):
     for more information on the message format.
 
     :keyword data_body: The body consists of one or more data sections and each section contains opaque binary data.
-    :paramtype data_body: Union[str, bytes, List[Union[str, bytes]]]
+    :paramtype data_body: str, bytes, or list[str or bytes]
```

Comment:
just wondering, but is this the right syntax and not:

    `:paramtype data_body: str or bytes or list[str or bytes]`